### DISCUSSION TITLE
could not compile `stylist-macros`

### DISCUSSION THREAD

**Discussion by Hojun-Son** [emanuele-em/proxelar#66] [Q&A] [answered]
__wbindgen_realloc: undefined version:
__wbindgen_malloc: undefined version:
__wbindgen_free: undefined version:
__wbindgen_exn_store: undefined version:
failed to set dynamic section sizes: bad value
          collect2: error: ld returned 1 exit status

How can I resolve it?

**Comment by emanuele-em** [comment, marked as answer]
I will definitely take a look in the next few days
